README.md

```diff
@@ -6,20 +6,23 @@ Exercism exercises in JavaScript
 
 To run the tests, you'll need NodeJS and Jasmine. For information about how to install these tools, see the [Javascript](http://exercism.io/languages/javascript/about) page.
 
-## Running Unit Test Suite
+## Tasks
 
-The following commands assume that you are in the 'javascript' directory:
+The following commands assume that you are in the `javascript` directory:
 
-### All Assignments
+### Unit Tests: All Assignments
 
-    % make test
+    make test
 
-### Single Assignment
+### Unit Tests: Single Assignment
 
-    % make test-assignment ASSIGNMENT=wordy
+    make test-assignment ASSIGNMENT=wordy
+
+### Code Style
+
+    yarn lint
```

Comment:
I think we agree on favoring `npm` over `yarn`. But it'll be interesting to allow contributors to use `yarn`. This doc could be like:

> Lint the code with `npm run lint`. If you use `yarn`, the command is `yarn lint`.

Reply:
I believe Yarns users are aware that it's possible to run npm scripts with the `yarn` executable. Someone who's not using the official Node.js package manager must have solid reasons [and comprehension of them] to be using it. Besides that, it's impossible for a contributor to have `yarn` but not `npm` available, since npm is bundled in Node.js.

Do you really think that a mention to Yarn in docs is something that worth keeping, @rchavarria?

Reply:
I don't have a strong feeling about it. I think we all agree that `npm` should be the tool to be exposed in the docs. `yarn` seems to be quite spread. I don't mind if the docs make some reference to it, like:

>  If you prefer, you can use `yarn` :wink: :wink: